src/back/shared/errors.hpp

```diff
@@ -1,11 +1,35 @@
 #pragma once
 
-#include <stdexcept>
+#include <userver/components/component_config.hpp>
+#include <userver/components/component_context.hpp>
+#include <userver/server/handlers/http_handler_json_base.hpp>
+#include <userver/utest/using_namespace_userver.hpp>
```

Comment:
А для чего тут всё это? Разве не достаточно:
```suggestion
#include <stdexcept>
#include <userver/server/http/http_status.hpp>
#include <userver/utest/using_namespace_userver.hpp>
```

Reply:
Готово (подчистил немного по зависимостям).